run_conode.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=1.0-pre1
```

Comment:
I guess it's possible to make onet.Server prints out the version or stg ? Or conode binary ?